package.json

```diff
@@ -22,9 +22,21 @@
   },
   "homepage": "https://github.com/vtex/eslint-config-vtex#readme",
   "dependencies": {
+    "@typescript-eslint/eslint-plugin": "^1.4.2",
+    "@typescript-eslint/parser": "^1.4.2",
     "eslint-config-prettier": "^4.0.0",
     "eslint-plugin-import": "^2.15.0",
     "eslint-plugin-lodash": "^5.1.0",
     "eslint-plugin-prettier": "^3.0.1"
+  },
+  "prettier": {
+    "semi": false,
+    "trailingComma": "es5",
+    "singleQuote": true
+  },
+  "devDependencies": {
+    "eslint": "^5.15.1",
+    "prettier": "^1.16.4",
+    "typescript": "^3.3.3333"
```

Comment:
is this correct?

Reply:
[yep](https://www.npmjs.com/package/typescript/v/3.3.3333).